articles/quickstart/native/flutter/_configure_urls_interactive.md

```diff
@@ -0,0 +1,33 @@
+## Configure Auth0 {{{ data-action=configure }}}
+
+To use Auth0 services, you need to have an application set up in the Auth0 Dashboard. The Auth0 application is where you will configure how you want authentication to work for your project.
+
+:::note
+The URLs below make use of a `SCHEME` placeholder, and this value varies depending on what platform you're working with. On Android, this can be `https` or some other custom scheme. On iOS, this is your application's [bundle identifier](https://developer.apple.com/documentation/appstoreconnectapi/bundle_ids).
+:::
+
+### Configure an application
+
+Use the interactive selector to create a new Auth0 application or select an existing application that represents the project you want to integrate with. Every application in Auth0 is assigned an alphanumeric, unique client ID that your application code will use to call Auth0 APIs through the SDK.
+
+Any settings you configure using this quickstart will automatically update for your application in the <a href="${manage_url}/#/">Dashboard</a>, which is where you can manage your applications in the future.
+
+If you would rather explore a complete configuration, you can view a sample application instead.
+
+### Configure Callback URLs
+
+A callback URL is a URL in your application that you would like Auth0 to redirect users to after they have authenticated. If not set, users will not be returned to your application after they log in.
+
+::: note
+If you are following along with our sample project, set this to `SCHEME://${account.namespace}/flutter/YOUR_PACKAGE_ID/callback`
```

Comment:
We can't use `/flutter/` because the underlying SDKs will use `/android/` and `ios`.

Reply:
Yes, exactly.

Reply:
I've correct this here: 5d55dc50d

Can you check it makes sense?

Reply:
I just took the liberty to replace `YOUR_PACKAGE_ID` with `YOUR_PACKAGE_NAME` (Android). I've approved the PR now, excellent work! Thanks for being patient with the review.